monitors/index.js

```diff
@@ -194,7 +202,13 @@ const Poll = async function(callback) {
     if (!process.env.EMP_ADDRESS) {
       throw "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party.";
     }
-    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 300000;
+
+    // Default to 1 minute delay. If set to 0 in env variables then the script will exit after full execution.
+    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 60 * 1000;
+
+    // Block number to search for events from. If set, acts to offset the search to ignore events in the past. If not
+    // set then default to null which indicates that the bot should start at the current block number.
+    const startingBlock = process.env.STARTING_BLOCK_NUMBER ? process.env.STARTING_BLOCK_NUMBER : null;
```

Comment:
You can just write `startingBlock = process.env.STARTING_BLOCK_NUMBER` since it is `null` by default